Formula/gnome-latex.rb

```diff
@@ -24,7 +31,14 @@ class GnomeLatex < Formula
   depends_on "libgee"
   depends_on "tepl"
 
+  # Add commit to port to Tepl 6
+  patch do
+    url "https://gitlab.gnome.org/Archive/gnome-latex/-/commit/e1b01186f8a4e5d3fee4c9ccfbedd6d098517df9.diff"
+    sha256 "0d54059732cb3092f52bfb8bca6ebad24a08b86036baafb31e06aca2415517ca"
+  end
+
   def install
+    system "autoreconf", "-fvi"
```

Comment:
`gnome-latex` was archived/deprecated without another release.

I currently added the unreleased commit for `tepl 6` to source tar.
The problem is that `configure` needs to be regenerated.
The official way is to use `./autogen.sh`: https://gitlab.gnome.org/Archive/gnome-latex/-/blob/master/autogen.sh
However, that is only available in the Git repo. It appears to do:
```sh
aclocal --install || exit 1
gtkdocize --copy || exit 1
intltoolize --force --copy --automake || exit 1
autoreconf --verbose --force --install || exit 1
```
Not too sure if the other 3 are necessary when rebuilding existing `configure`.
For now, trying build with just `autoreconf` step.

As reference, ArchLinux has chosen to use Git repo directly pointing to final commit in master branch.
https://github.com/archlinux/svntogit-community/blob/packages/gnome-latex/trunk/PKGBUILD#L17